src/pdfjsWrapper.js

```diff
@@ -188,8 +188,7 @@ export default function(PDFJS) {
 			if ( pdfPage === null )
 				return;
 
-			if ( rotate === undefined )
-				rotate = pdfPage.rotate;
+			rotate = (typeof pdfPage.rotate === 'undefined'? 0 : pdfPage.rotate) + (typeof rotate === 'undefined'? 0 : rotate);
```

Comment:
> `typeof pdfPage.rotate === 'undefined'`

why not just `pdfPage.rotate === undefined` ?

Reply:
I'll change it to pdfPage.rotate === undefined
and rotate === undefined

typeof pdgPage.rotate === 'undefined' was how I learned to do it from older StackOverflow answers.

Reply:
Fixed now.